src/main/java/org/folio/consortia/domain/dto/Permission.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.consortia.domain.dto;
+
+import lombok.Data;
+
+@Data
```

Comment:
this module uses java 17, probably java record will be sufficient here?

Reply:
fixed.